chef/cookbooks/utils/libraries/helpers.rb

```diff
@@ -40,12 +40,12 @@ def self.get_host_for_public_url(node, use_ssl, use_cluster = false)
 
       # Specify default as the configured public name
       public_name = CrowbarPacemakerHelper.cluster_haproxy_vpublic_name(node)
-      public_fqdn = "public.#{cluster_vhostname}.#{node[:domain]}"
+      public_fqdn = "public-#{cluster_vhostname}.#{node[:domain]}"
       public_net_db = Chef::DataBagItem.load("crowbar", "public_network").raw_data
       public_ip = public_net_db["allocated_by_name"]["#{cluster_vhostname}.#{node[:domain]}"]["address"]
     else
       public_name = node[:crowbar][:public_name]
-      public_fqdn = "public."+node[:fqdn]
+      public_fqdn = "public-"+node[:fqdn]
```

Comment:
Style/SpaceAroundOperators: Surrounding space missing for operator `+`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)